microshift_cli_ref/microshift-cli-using-oc.adoc

```diff
@@ -0,0 +1,82 @@
+:_content-type: ASSEMBLY
+[id="microshift-using-oc"]
+= Using the `oc` tool
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-using-oc
+
+toc::[]
+
```

Comment:
According to the [Modular documentation guidelines](https://redhat-documentation.github.io/modular-docs/#assembly-guidelines), an introductory sentence is required for the assembly.

Applies to the `microshift_cli_ref/microshift-oc-cli-install.adoc` and `microshift_cli_ref/microshift-oc-config.adoc` assembles as well.